### PR TITLE
First structure shell of README file

### PR DESCRIPTION
**Pull request by vcuspinera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+### Contributors:
+|Name|GitHub|
+|----|-------------|
+|Ying Jing -Jacky- Ho |[jackyho112](https://github.com/jackyho112)|
+|Ryan Homer |[ryanhomer](https://github.com/ryanhomer)|
+|Derek Kruszewski |[dkruszew](https://github.com/dkruszew)|
+|Victor Cuspinera |[vcuspinera](https://github.com/vcuspinera)|
+
+### Overview:
+If you have face a database with so many features that while working it gets messy, a good idea is to select some of these features for your model. However, this is not a trivial question, and to help you with this task, we have created the `featureselection` package.
```
Comment (techrah) [inline README.md:14]: I think we need to work on this some more but for the time being, it's a good placeholder.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+### Feature description:
+In this package, four functions are included to lead with feature selection:
+
+#### Forward
+- TO DESCRIBE
+
+#### Backward
+- TO DESCRIBE
+
+#### Function 3
```
Comment (techrah) [inline README.md]: This one should be "Variance Thresholding"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#### Forward
+- TO DESCRIBE
+
+#### Backward
+- TO DESCRIBE
+
+#### Function 3  
+- TO DESCRIBE
+
+#### Function 4
```
Comment (techrah) [inline README.md]: This one should be "Simulated Annealing"

**Review by techrah — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vcuspinera) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (vcuspinera) [reply README.md]: Done

**Review by techrah — APPROVED** [review]

**Pull request merged**